### PR TITLE
Enable the switchless sample on Windows

### PR DESCRIPTION
**Pull request by jxyang**

### CODE REVIEW

**Comment by jxyang** [comment]
bors try

File: tests/switchless_threads/enc/enc.c [hunk 1]
```diff
@@ -70,6 +70,6 @@ OE_SET_ENCLAVE_SGX(
     1,    /* ProductID */
     1,    /* SecurityVersion */
     true, /* AllowDebug */
-    1024, /* HeapPageCount */
-    1024, /* StackPageCount */
+    128,  /* HeapPageCount */
```
Comment (radhikaj) [inline tests/switchless_threads/enc/enc.c]: @jxyang , why did this need to be changed?

Comment (jxyang) [reply tests/switchless_threads/enc/enc.c]: This test fails on Windows 2-core machines because it goes beyond the EPC limit (62mb?)

**Pull request merged**